protocol/transport/whisper_service.go

```diff
@@ -78,14 +76,14 @@ type WhisperServiceTransport struct {
 
 var _ WhisperTransport = (*WhisperServiceTransport)(nil)
 
-type StatusNode interface {
-	GethNode() *gethnode.Node
+type Server interface {
```

Comment:
Does it need to be exported?

Reply:
other module can do common check to ensure that their object implements interface, but it is very unlikely i guess